newapi/ooniapi/measurements.py

```diff
@@ -1166,6 +1166,16 @@ def get_aggregated() -> Response:
         description: >-
           The end date of when measurement were run (ex.
           "2016-10-20T10:30:00")
+      - name: time_gran
+        in: query
+        type: string
+        description: Time granularity
+        enum:
+        - hour
+        - day
+        - week
+        - year
```

Comment:
Can we add month here? If it's too many aggregation, I would suggest dropping `year` in favour of `month` as that is a much more common use-case.

Reply:
Fixed